src/vxworks/mod.rs

```diff
@@ -463,6 +474,46 @@ cfg_if! {
                     .finish()
             }
         }
+
+        impl PartialEq for sa_u_t {
+            fn eq(&self, other: &sa_u_t) -> bool {
+                unsafe { self.sa_handler == other.sa_handler }
+            }
+        }
+        impl Eq for sa_u_t {}
+        impl ::fmt::Debug for sa_u_t {
+            fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
+                f.debug_struct("sa_u_t")
+                    .field("sa_handler", unsafe { &self.sa_handler })
+                    .field("sa_sigaction", unsafe { &self.sa_sigaction })
```

Comment:
Not sure what the "best practice" is for implementing Debug for unions. Other implementations appear to only print one of the fields (or even none of the fields), but attempting to print all the variants seems reasonable too (even though only one of the variants is meaningful at any time).